src/database/prisma.service.ts

```
@@ -0,0 +1,36 @@
+import { Injectable, OnModuleDestroy, OnModuleInit } from '@nestjs/common';
+import { PrismaClient } from '@prisma/client';
+import { readReplicas } from '@prisma/extension-read-replicas';
+import 'dotenv/config';
+
+@Injectable()
+export class PrismaService implements OnModuleInit, OnModuleDestroy {
+  private _client: ReturnType<typeof this.createExtendedClient>;
+
+  constructor() {
+    this._client = this.createExtendedClient();
+    return Object.assign(this, this._client);
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)